apps/dashboard/src/contract-ui/tabs/account/components/deposit-native.tsx

```diff
@@ -46,6 +46,7 @@ export const DepositNative: React.FC<DepositNativeProps> = ({
         value={amount}
       />
       <TransactionButton
+        txChainID={v5Chain.id}
```

Comment:
given there is no `v4Chain` anymore anywhere (or there shouldn't be) can we re-name to `chain`?

Reply:
We still have `Chain` and `ChainMetadata` (v4, v5) chain objects everywhere. The all-chains data is of type `ChainMetadata` (v4)